ADOPTERS.md

```diff
@@ -0,0 +1,21 @@
+# Paketo Buildpacks Adopters
+
+If you're using Paketo Buildpacks and want to add your organization to this list, just add a response to this Discussion and we'll get your organization added.
```

Comment:
Was there supposed to be a link for `this Discussion`?

Reply:
Yes, it is fixed